cypress-shared/integration/adyen/2.1-graphql_testcases.spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jest/expect-expect */
+import {
+  graphql,
+  createAccountHolder,
+  validateCreateAccountHolderResponse,
```

Comment:
@shashidhar85 ensure security check gets passed in pr
lint, sonarcloud will run only after security check gets passed 
so, please do it asap